src/translations/en.json

```diff
@@ -46,8 +46,8 @@
   "medical.serviceLevel.null": "None",
   "medical.serviceLevel.S": "Simple Service",
   "medical.serviceLevel.V": "Visit",
-  "medical.serviceLevel.D": "Day Hospital",
-  "medical.serviceLevel.H": "Hospital Care",
+  "medical.serviceLevel.D": "Day of Stay",
+  "medical.serviceLevel.H": "Hospital Case",
```

Comment:
Did you also change the translations in Lokalise?

Reply:
Done